<?xml version="1.0" ?>
<launch>




  <group ns="ds1">
    <param name="input_rob_pose_topic_name" value="/lwr/ee_pose"/>
    <param name="input_rob_vel_topic_name" value="/lwr/ee_vel"/>
    <param name="input_rob_acc_topic_name" value="/lwr/ee_accel"/>
    <param name="input_rob_force_topic_name" value="/lwr/ee_ft"/>
    <param name="output_vel_topic_name" value="/lwr/joint_controllers/passive_ds_command_vel"/>
    <param name="output_filtered_vel_topic_name" value="/filtered_output"/>
    <rosparam command="load" file="$(find adaptive_polishing)/config/polishing_init_param.yml"/>   
    <node name="main_node" pkg="adaptive_polishing" type="adaptive_polishing_node" output="screen"/>
    <!-- <node name="TA_record" pkg="adaptive_polishing" type="adaptation_recorder_node" output="screen"/> -->
  </group>

  <node pkg="rostopic" type="rostopic" name="angle_setting" args="pub /lwr/joint_controllers/passive_ds_command_orient geometry_msgs/Quaternion '{x: 0.0, y: 1.0, z: 0.0, w: 0.0}' -r 100" />

  <node name="rqt_reconfigure" pkg="rqt_reconfigure" type="rqt_reconfigure" />

</launch>